x-pack/plugins/cases/common/constants.ts

```diff
@@ -77,8 +71,7 @@ export const SUPPORTED_CONNECTORS = [
 /**
  * Alerts
  */
-export const MAX_ALERTS_PER_SUB_CASE = 5000;
-export const MAX_GENERATED_ALERTS_PER_SUB_CASE = 50;
+export const MAX_ALERTS_PER_CASE = 5000;
```

Comment:
Why need that?

Reply:
We use it in the alert service to limit how many alert we retrieve I think for the connectors maybe?

https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/server/services/alerts/index.ts#L186